cfgov/v1/tests/test_util_migrations.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import mock
+from unittest import TestCase
+
+from v1.util.migrations import (
+    migrate_page_types_and_fields,
+    migrate_stream_field,
+    get_stream_data,
+    set_stream_data,
+)
+
+
+class MigrationsUtilTestCase(TestCase):
```

Comment:
I want to come back to this because there was some interesting discovery that I made in the process of using an actual Page and PageRevision object to test these functions with, that I hope we can apply in future too.

Because these functions operate on StreamFields, I needed a Page subclass with a StreamField. We clearly have multiple pages like that in `v1.models`, but all of them are more complex than I really needed for testing purposes. Wagtail uses [`wagtail.tests.testapp`](https://github.com/torchbox/wagtail/tree/master/wagtail/tests/testapp) for its tests, and I started digging into it. It includes [StreamPage](https://github.com/torchbox/wagtail/blob/master/wagtail/tests/testapp/models.py#L634) which is a perfect page type for testing these functions: it's a light-weight, direct subclass of Page with an added StreamField.

I want to call attention to this because there are other nice little models in `testapp` too that might be useful to us in other testing scenarios.